AsaApi/Core/Private/Tools/Requests.cpp

```diff
@@ -279,6 +362,49 @@ namespace API
 		return true;
 	}
 
+	bool Requests::CreatePostRequest(const std::string& url, const std::function<void(bool, std::string, std::unordered_map<std::string, std::string>)>& callback,
```

Comment:
Can we reduce duplicate code, most of this seems to be a similar implementation to another CreatePostRequest call.